lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -60,19 +53,33 @@ $-separator-color: var(--nx-swatch-indigo-40);
   color: $-text-color;
   flex: none;
   margin: 0;
+  position: relative;
 
   &:focus:not(:active) {
-    outline-color: $-focus-color;
-    padding-top: calc(var(--nx-spacing-base) - #{$-button-border-width});
-    padding-bottom: calc(var(--nx-spacing-base) - #{$-button-border-width});
+    outline-color: var(--nx-color-global-sidebar-border-focus);
   }
 
   &:hover {
-    background-color: $-hover-color;
+    background-color: var(--nx-color-global-sidebar-toggle-background-hover);
+    .nx-icon {
+      color: var(--nx-color-global-sidebar-toggle-color);
+    }
+  }
+
+  &:hover:focus:not(:active)::before {
```

Comment:
Wasn't this implemented differently in an earlier iteration of this PR? What prompted the change?

Reply:
> Wasn't this implemented differently in an earlier iteration of this PR? What prompted the change?

I'm afraid not. This was the implemented from the start.